### PR TITLE
server: forget to add `StmtCommit` when loading data

### PR DESCRIPTION
**Pull request by zimulala**
The operation of `load data` divides into several batches. Every batch needs to commit once. So it also needs to call `StmtCommit`.

Relate to #5678

### CODE REVIEW

File: server/conn.go [hunk 1]
```diff
@@ -729,6 +729,7 @@ func insertDataWithCommit(goCtx goctx.Context, prevData, curData []byte, loadDat
 		if !reachLimit {
 			break
 		}
+		terror.Log(loadDataInfo.Ctx.StmtCommit())
```
Comment (shenli) [inline server/conn.go]: If StmtCommit meets error, should we return the error?

Comment (coocood) [reply server/conn.go]: Other places only log the StmtCommit error too.
We can discuss how to treat this error later.

**Comment by coocood** [comment]
LGTM

File: server/conn.go (hunk 1 above)
Comment (tiancaiamao) [reply server/conn.go]: Just `terror.Log` it now, I'll handle it later. @shenli

Comment (shenli) [reply server/conn.go]: ok

File: server/server_test.go [hunk 2]
```diff
@@ -322,6 +323,8 @@ func runTestLoadData(c *C, server *Server) {
 		"xxx row5_col1	- 	row5_col3")
 	c.Assert(err, IsNil)
 
+	// If the MemBuffer can't be committed once in each batch, it will return an error like "transaction is too large".
+	kv.TxnTotalSizeLimit = 10240
```
Comment (tiancaiamao) [inline server/server_test.go]: Recover it to default value after this test?

**Comment by zimulala** [comment]
PTAL @tiancaiamao

**Comment by tiancaiamao** [comment]
LGTM

**Pull request merged**